### PR TITLE
patch method added

### LINKED ISSUE #40: Add PATCH method
Add `PATCH` method to `/users` endpoint.

The method **should not** allow the request to manipulate any data on the database. Instead, the data sent by the user should be returned exactly as it is, with an additional field `timestamp` with the current timestamp, and a positive response code `200`

### PR DESCRIPTION
**Pull request by subhangi2731**
fixes #40 patch method added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,9 +20,10 @@
   },
   "homepage": "https://github.com/adityabisoi/fetch#readme",
   "dependencies": {
+    "dateformat": "^4.5.1",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.10.9",
+    "mongoose": "^5.12.1",
```
Comment (ShauryaAg) [inline package.json:26]: Why did you change the mongoose version?

**Review by ShauryaAg — CHANGES_REQUESTED** [review]
Don't commit your `package-lock.json`

**Comment by adityabisoi** [comment]
Also, the method has hardcoded fields. We must return anything that is received, with a timestamp.

**Comment by subhangi2731** [comment]
I will update the pr now @adityabisoi

**Pull request merged**